config/php-cs-fixer.php

```diff
@@ -20,30 +19,31 @@
             '@PHPUnit57Migration:risky' => true,
             '@PHPUnit60Migration:risky' => true,
             '@PHPUnit75Migration:risky' => true,
-            '@PSR2' => true,
+            '@PSR12' => true,
             'array_syntax' => ['syntax' => 'short'],
             'blank_line_after_opening_tag' => true,
             'cast_spaces' => ['space' => 'none'],
+            'class_attributes_separation' => true,
             'combine_consecutive_issets' => true,
             'combine_consecutive_unsets' => true,
             'compact_nullable_typehint' => true,
             'concat_space' => ['spacing' => 'one'],
             'declare_equal_normalize' => true,
             'declare_strict_types' => true,
             'dir_constant' => true,
+            'echo_tag_syntax' => true,
             'elseif' => true,
             'encoding' => true,
             'escape_implicit_backslashes' => ['single_quoted' => true],
             'function_typehint_space' => true,
-            'hash_to_slash_comment' => true,
             'is_null' => true,
             'linebreak_after_opening_tag' => true,
             'lowercase_cast' => true,
             'magic_constant_casing' => true,
-            'method_separation' => true,
             'modernize_types_casting' => true,
+            'multiline_whitespace_before_semicolons' => true,
             'native_function_casing' => true,
-            'native_function_invocation' => true,
+            'native_function_invocation' => ['include' => ['@internal']],
```

Comment:
To keep the previous behaviour, we should probably have `@all` rather than `@internal`.  Though we don't and are unlikely ever to have any global functions, it's possible we may use some from a third-party library.

Reply:
Done.